ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -29,6 +29,7 @@ def test_numpy_mean(
     fn_tree,
     on_device,
     keep_dims,
+    rtol=1e-2,
```

Comment:
you have added it in the function definition, it should not be added here, it should just be passed where we call the `test_function(_, rtol=somevalue,_)` at the end